env/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  acropolis-mysql:
+    image: 'mysql:8'
+    environment:
+      - MYSQL_ROOT_PASSWORD=root
+      - MYSQL_PASSWORD=root
+      - MYSQL_DATABASE=acropolisdev
+    ports:
+      - 3306:3306
```

Comment:
Might also want to bind a data volume, to persist things betweens composes, and would make startup a bit faster. But is a minor detail, can always add that later.

Reply:
True, for now the container is independent of the application so it can stay alive for your development session. Can be added later I think